src/action.ml

```diff
@@ -200,7 +200,8 @@ module Var_expansion = struct
     | Paths   (l, Concat) -> [concat (List.map l ~f:(string_of_path ~dir))]
 
   let to_string ~dir = function
-    | Strings (_, Split) | Paths (_, Split) -> assert false
+    | Strings (_, Split) | Paths (_, Split) ->
+       failwith "Action.Var_expansion.to_string"
```

Comment:
Given that this wants to be matched on later, I think it would be better to define an exception, rather than use `Failure`